servant-mock/servant-mock.cabal

```diff
@@ -0,0 +1,46 @@
+name:                servant-mock
+version:             0.4
```

Comment:
Can we have the same version number as everything else? (It'll get bumped soon enough, to be fair...)